### PR TITLE
Create karma_geoff.md

### PR DESCRIPTION
**Pull request by JJM317**
Sorry my tags are weak. I really cannot come up with anything creative here.

Also, this looks like it comes from a book of collections and is also available on Thanissaro Bhikkhu's website dhammatalks.org. Do you want to use either of these instead of access to insight?

### CODE REVIEW

File: _content/essays/karma_geoff.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Karma"
+authors:
+  - geoff
+external_url: "https://www.accesstoinsight.org/lib/authors/thanissaro/karma.html"
```
Comment (buddhist-uni) [inline _content/essays/karma_geoff.md:5]: Yeah, AtI `source` and dhammatalks.org `external` URLs are preferred for Than Geoff's essays 😁 Thanks

File: _content/essays/karma_geoff.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+drive_links:
+  - "https://drive.google.com/file/d/1KNwJ3NH-8HoP0huE7aO0UQHVCNBLdXR5/view?usp=sharing"
+formats: [pdf]
+course: view 
+tags:
+  - karma
+  - effort
```
Comment (buddhist-uni) [inline _content/essays/karma_geoff.md:12]: Yeah, just one tag for "karma" here is good enough. While most items will get 4 or 5 tags, there are some that even get zero if they're short and really only about one thing 😁

File: _content/essays/karma_geoff.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+year: 2000
+pages: "3"
```
Comment (buddhist-uni) [inline _content/essays/karma_geoff.md:14]: page counts not in quotes please

**Review by buddhist-uni — COMMENTED** [review]
Naw, essays (and not mentioning the book) is good for this one 😁

**Comment by buddhist-uni** [comment]
Great! And the urls?

**Comment by JJM317** [comment]
What does 'and the urls?' mean? I have the external link and the drive
link. Is the external link incorrect?
--------------------

**Comment by buddhist-uni** [comment]
I mean, let's do:

```yaml
external_url: "https://www.dhammatalks.org/books/NobleStrategy/Section0005.html"
source_url: "https://www.accesstoinsight.org/lib/authors/thanissaro/karma.html"
```

**Comment by buddhist-uni** [comment]
Oh, and since this is in "Noble Strategy" which is in the library, let's maybe move this to `excerpts` and add:

```yaml
from_book: noble-strategy_geoff
chapter: 5
```

**Review by buddhist-uni — APPROVED** [review]
Looks good, thanks! 😁

**Pull request merged**